spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -14,37 +14,10 @@ bom {
 			issueLabels = ["type: dependency-upgrade"]
 		}
 	}
-	library("ActiveMQ", "6.1.2") {
+	library("ActiveMQ", "6.1.3") {
 		group("org.apache.activemq") {
-			modules = [
-					"activemq-amqp",
-					"activemq-blueprint",
-					"activemq-broker",
-					"activemq-client",
-					"activemq-console" {
-						exclude group: "commons-logging", module: "commons-logging"
```

Comment:
Did you check this is no longer necessary?

Reply:
Hi @snicoll, those are still necessary. How should I proceed?

Reply:
We can use the bom and then also declare the modules that require exclusions. Here's an example for Micrometer:

```
	library("Micrometer", "1.9.17") {
		considerSnapshots()
		group("io.micrometer") {
			modules = [
				"micrometer-registry-stackdriver" {
					exclude group: "javax.annotation", module: "javax.annotation-api"
				}
			]
			imports = [
				"micrometer-bom"
			]
		}
	}
```

This uses `micrometer-bom` while also adding an exclusion to the `micrometer-registry-stackdriver` dependency that it manages.

Reply:
Thanks!